beaker-vue/src/components/BeakerNotebook.vue

```diff
@@ -75,58 +21,29 @@
                     :size="70"
                     :minSize="30"
                     class="main-panel"
-                    @keydown="handleKeyboardShortcut"
```

Comment:
Need to think on communication with handling keyboard shortcuts from the right parent, as using only under <Notebook /> component is limiting (need to click on cells in order to use keyboard shortcuts).

Reply:
As of right now the keyboard shortcuts do respond to the same container as before, but it requires using a ref/expose combo and doesn't sounds like a clean interface for a reusable vue component. Need to think about this some more.